environment.py

```diff
@@ -213,12 +213,12 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DSS_WORKERS': '8',
 
         # Whether to create a subscription to DSS during deployment. Set this
-        # variable to 1 to enable `make deploy` to subscribe the indexer in the
+        # variable to 1 to enable `make subscribe` to subscribe the indexer in the
```

Comment:
Good catch.

Reply:
@jessebrennan and I were able to address this together.